roles/openshift_health_checker/openshift_checks/disk_availability.py

```diff
@@ -121,11 +122,21 @@ def run(self):
 
         return {}
 
+    def find_ansible_submounts(self, path):
+        """Return a list of ansible_mounts that are below the given path."""
+        base = os.path.join(path, "")
```

Comment:
If `path` were `/`, this would set `base` to `/`, thus returning `/` as a submount, which would cause `self.free_bytes('/')` to count the `/` mount twice, but I guess we never call `self.free_bytes('/')`, so it is fine even if not strictly correct.

Reply:
It's true. If we ever have a check on the size of `/` ...